src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    private final Random randomColor = new Random();
+
     public String getRandomColor() {
```

Comment:
Consider adding a null check for an empty array of colors in the getRandomColor() method. If you attempt to access an element of an empty array, it will cause an ArrayIndexOutOfBoundsException.

Reply:
I think I've done this correctly but the Inteliij idicate warning about "colors == null is always false.
Should I do it differently?